buildScript/webpack.config.js

```diff
@@ -137,7 +147,12 @@ export default function makeWebpackConfig(config) {
                         ['@babel/preset-env',
                             {
                                 targets: {
-                                    browsers: ['safari >= 16', 'chrome >= 130', 'firefox >= 134', 'edge >= 130']
+                                    browsers: [
+                                        'safari >= '+MIN_SAFARI_VERSION,
+                                        'chrome >= '+MIN_CHROME_VERSION,
+                                        'firefox >= '+MIN_FIREFOX_VERSION,
+                                        'edge >= '+MIN_EDGE_VERSION
+                                    ],
```

Comment:
Manually pinning specific versions is fine. However, if we’d prefer Babel to enforce a standard, we can rely on its built-in target policies instead.  e.g.
```
browsers: 'baseline widely available'
browsers: 'last 2 years'
```